### PR TITLE
Add draft functionality

### PR DESCRIPTION
**Pull request by maxmoeschinger**

### CODE REVIEW

File: sourcecode/proxies/lti/src/services/lti.js [hunk 1]
```diff
@@ -137,6 +137,7 @@ const editResourceRequest = async (
     reqProtoHost,
     language
 ) => {
+    console.log('asdf');
```
Comment (emmachughes) [inline sourcecode/proxies/lti/src/services/lti.js:140]: I feel log messages should be more descriptive than this.

File: sourcecode/proxies/lti/src/services/lti.js [hunk 2]
```diff
@@ -148,6 +149,8 @@ const editResourceRequest = async (
         resourceVersionId
     );
 
+    console.log('asdf 2');
```
Comment (emmachughes) [inline sourcecode/proxies/lti/src/services/lti.js:152]: Ditto

File: sourcecode/apis/contentauthor/app/Http/Controllers/ArticleController.php [hunk 3]
```diff
@@ -334,6 +336,7 @@ public function update(ArticleRequest $request, Article $article)
         $article->max_score = $article->getMaxScoreHelper($article->content);
         $article->license = $request->input('license', $oldLicense);
         $article->is_published = $article::isDraftLogicEnabled() ? $request->input('isPublished', 1) : 1;
+        $article->is_draft = $request->input('isDraft', 0);
```
Comment (emmachughes) [inline sourcecode/apis/contentauthor/app/Http/Controllers/ArticleController.php:339]: `false` should be the default value here, not `0`.

File: sourcecode/apis/contentauthor/database/migrations/2022_03_17_125309_add_is_draft.php [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+class AddIsDraft extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('h5p_contents', function (Blueprint $table) {
```
Comment (emmachughes) [inline sourcecode/apis/contentauthor/database/migrations/2022_03_17_125309_add_is_draft.php:16]: All of these should have `false` as their defaults. The fact that booleans map to `TINYINT(1)` is an implementation detail we should not concern ourselves with, Laravel will do the appropriate conversion.

**Review by emmachughes — CHANGES_REQUESTED** [review]

File: sourcecode/apis/contentauthor/database/migrations/2022_03_17_125309_add_is_draft.php (hunk 4 above)
Comment (maxmoeschinger) [reply sourcecode/apis/contentauthor/database/migrations/2022_03_17_125309_add_is_draft.php:16]: agreed

**Pull request merged**